appt-org/appt-website
# Add remarks about EditText role

# What does this PR do?
After spending a few hours looking through the documentation, found out this non-obvious and undocumented use case. Though it would be worth adding a section about it.

## How do you test this PR?

Test link: https://appt.org/en/docs/android/samples/accessibility-role

Test steps:

Go to https://appt.org/en/docs/android/samples/accessibility-role and verify a new section at the bottom.

## Checklist

- [ ] Works on Safari, Chrome, Firefox & Edge
- [ ] Works on different screen sizes (responsive)
- [ ] Scores 100 in [Lighthouse/PageSpeed Insights](https://pagespeed.web.dev) on SEO, accessibility and best practices, and >= 90 on performance
- [ ] Does not contain tracking cookies
- [ ] Are migrations added?
- [ ] Is the README.md updated?

## Accessibility checklist

- [ ] Scalable text
- [ ] Dark and light mode
- [ ] Focus management
- [ ] Aria attributes
